packages/editor/src/components/dialogs/CreatePrefabPanelDialog.tsx

```diff
@@ -68,99 +74,168 @@ export default function CreatePrefabPanel({ entity }: { entity: Entity }) {
-          query: { key: 'projects/' + srcProject + '/' + fileName }
-        })
-        if (resources.data.length === 0) {
-          throw new Error('User not found')
+        if (isExportLookDev) {
+          const lookdevEntity = [] as Entity[]
+          const lookDevComponent: Component[] = [
+            SkyboxComponent,
+            HemisphereLightComponent,
+            DirectionalLightComponent,
+            PostProcessingComponent
+          ]
+          const prefabEntity = createEntity()
+          const obj = new Scene()
+          addObjectToGroup(prefabEntity, obj)
+          proxifyParentChildRelationships(obj)
+          const rootEntity = getState(EditorState).rootEntity
+          iterateEntityNode(rootEntity, (entity) => {
+            lookDevComponent.forEach((component) => {
+              if (hasComponent(entity, component)) {
+                lookdevEntity.push(entity)
+              }
+            })
+          })
+          setComponent(prefabEntity, EntityTreeComponent, { parentEntity: rootEntity })
+          setComponent(prefabEntity, NameComponent, 'temp prefab')
+          lookdevEntity.forEach((entity) => {
+            setComponent(entity, EntityTreeComponent, { parentEntity: prefabEntity })
+          })
+
+          getMutableState(SelectionState).selectedEntities.set([])
+
+          await exportRelativeGLTF(prefabEntity, srcProject, fileName)
+
+          const resources = await API.instance.service(staticResourcePath).find({
+            query: { key: 'projects/' + srcProject + '/' + fileName }
+          })
+          if (resources.data.length === 0) {
+            throw new Error('User not found')
+          }
+          const resource = resources.data[0]
+          const tags = [...prefabTag.value]
+          tags.push('Lookdev')
+          await API.instance.service(staticResourcePath).patch(resource.id, { tags: tags, project: srcProject })
+          setComponent(prefabEntity, NameComponent, 'temp prefab')
+
+          lookdevEntity.forEach((entity) => {
+            setComponent(entity, EntityTreeComponent, { parentEntity: rootEntity })
+          })
+          removeEntity(prefabEntity)
+          await EditorControlFunctions.removeObject(lookdevEntity)
+          lookdevEntity.forEach((entity) => {
+            removeEntity(entity)
+          })
+          addMediaNode(fileURL)
+            .catch((error) => getMutableState(EditorErrorState).error.set(error))
+            .then((uuid) => {
+              const entity = UUIDComponent.getEntityByUUID(uuid)
+              setComponent(entity, NameComponent, prefabName.value)
+            })
+          PopoverState.hidePopupover()
+          defaultPrefabFolder.set('assets/custom-prefabs')
+          prefabName.set('prefab')
+          prefabTag.set([])
+          isOverwriteModalVisible.set(false)
+          isOverwriteConfirmed.set(false)
+          PopoverState.showPopupover(<PrefabConfirmationPanelDialog entity={entity as Entity} />)
```

Comment:
this works, but put it in a helper function that you call at the end of both conditionals